src/ui/controllers/booking/routes.ts

```diff
@@ -11,36 +12,40 @@ router.param('courseId', asyncHandler(courseController.loadCourse))
 router.param('moduleId', asyncHandler(courseController.loadModule))
 router.param('eventId', asyncHandler(courseController.loadEvent))
 
-router.get('/book/ouch', bookingController.renderOuch)
+router.use((req: Request, res: Response, next: NextFunction) => {
+	if (!req.user.hasLineManager()) {
+		return res.redirect(`/profile/line-manager?redirectTo=${req.originalUrl}`)
```

Comment:
Refer to test doc; the only way users can hit these `/book` endpoints now is if they have saved them somewhere in bookmarks etc. The course overview page (that has the links to take the user to the booking page) will show the new message instead of the booking link, if the user does not have a line manager set.

Reply:
I saw the test doc and tested it myself locally. I just think it may surprise the user if the line manager page pops up without any context or warning. Regardless, all good!